ci/doc/how-to-setup-taos-ci-server.md

```diff
@@ -216,9 +216,9 @@ Finally, let's generate automatically PDF book per 1 hour with cron table (e.g.,
 $ sudo vi /etc/crontab
 # Generate doxygen document
 20 * * * * www-data cd /var/www/html/{your_prj_name}/ ; git pull
-30 * * * * www-data /var/www/html/{your_prj_name}/doc/book-hard-copy-prj-generate.sh
+30 * * * * www-data /var/www/html/{your_prj_name}/doc/book-doxygen-publish.sh
```

Comment:
Is it now `../doxybook/..` instead of `../doc/..`?

Reply:
Ah BTW, `/ci/` is also missing.

Reply:
@dongju-chae , Sure,  For maintenance and readability.

Reply:
> Ah BTW, /ci/ is also missing.

Nice catch. Fixed.